debian/README.source

```diff
@@ -2,12 +2,11 @@ Debian packaging of LinuxCNC/Mesaflash
 ======================================
 
 The following steps were performed to transform Upstream's git repository
-into a source tarball and a debian subdirectory that shines for the main
-distribution. The major obstacle is that Mesaflash is used across a wide
-span of distributions, from Wheezy  (Jessie anyon?) up to the latest
-Ubuntu releases. Not too many changes are required for Debian unstable,
-the most prominent one likely is that there is already a debiah directory
-in the source tree.
+into a source tarball and to provide a debian subdirectory that shines
```

Comment:
I'm not sure what "shines" means in this context.

Reply:
bump

Reply:
"looks very good" - let me eliminate this.